com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,202 @@
+{
+    "app-id": "com.ozmartians.VidCutter",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "vidcutter",
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "no-debuginfo": false,
+        "strip": true
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--device=dri",
+        "--talk-name=com.canonical.Unity.LauncherEntry",
+        "--talk-name=ca.desrt.dconf",
+        "--env=LC_NUMERIC=C",
```

Comment:
This is caused by a `fontconfig` regression that is fixed in the next release. Could you put that in a comment here for future.

Reply:
@TingPing 

which line in the quoted code are you referring to? because most of that is actually required by libmpv. The ca.desrt.dconf is for the Gnome D-Bus permissions (I know very little in this area but the app does talk to D-Bus for Taskbar process completion display which only works well in KDE really)

Reply:
`LC_NUMERIC`. Generally applications set that themselves since they know `libmpv` requires it. But `fontconfig` recently had a regression that overrides it. I assumed that is why this was here.

Reply:
@TingPing good to know but its actually for libmpv in this case as per client API notes at https://github.com/mpv-player/mpv/blob/master/libmpv/client.h regardsing LC_NUMERIC. Any Flatpak utilising mpv's client API would be setting this as well.